wordpress-mobile/WordPress-Android
# Issue/13268 add scrolling behavior to filter bar

Partially fixes #13268

This PR
- Fixes the scrolling behavior of the toolbar on Activity Log
- Removes hardcoded strings for Activity Log filters
- Adds horizontal scroll view to the filters bar -> not visible unless you have a really small screen 

To test:
1. Disabled ActivityLogFilters feature flag
2. Go to Activity Log
3. Make sure the filters are hidden
4. Scroll down and notice the toolbar disappears as you scroll and re-appears when you scroll up
--------------------
1. Enable ActivityLogFilters feature flag
2. Go to Activity Log
3. Make sure the filters are shown and the "Activity Range" and "Activity Type" texts appear
4. Scroll down and notice the filters bar remains fixed at the top


PR submission checklist:
